core/src/main/java/org/elasticsearch/search/highlight/HighlighterParseElement.java

```diff
@@ -79,7 +85,7 @@ public SearchContextHighlight parse(XContentParser parser, QueryShardContext que
 
         final SearchContextHighlight.FieldOptions.Builder globalOptionsBuilder = new SearchContextHighlight.FieldOptions.Builder()
                 .preTags(DEFAULT_PRE_TAGS).postTags(DEFAULT_POST_TAGS).scoreOrdered(false).highlightFilter(false)
-                .requireFieldMatch(true).forceSource(false).fragmentCharSize(100).numberOfFragments(5)
+                .requireFieldMatch(true).forceSource(false).fragmentCharSize(DEFAULT_FRAGMENT_CHAR_SIZE).numberOfFragments(DEFAULT_NUMBER_OF_FRAGMENTS)
                 .encoder("default").boundaryMaxScan(SimpleBoundaryScanner.DEFAULT_MAX_SCAN)
                 .boundaryChars(SimpleBoundaryScanner.DEFAULT_BOUNDARY_CHARS)
                 .noMatchSize(0).phraseLimit(256);
```

Comment:
Minor nit: Why no default constant for these to numbers and the two boolean parameters in line 88?